test/geometry/test_conversions.py

```diff
@@ -1381,6 +1381,153 @@ def test_jit(self, device, dtype):
         assert_close(actual, expected)
 
 
+class TestProjectPoints:
```

Comment:
```suggestion
class TestProjectPoints(BaseTester):
```

then you can use `self.assert_close` with handles some dtype tolerance amongst others